bears/general/FilenameBear.py

```diff
@@ -19,14 +20,20 @@ class FilenameBear(LocalBear):
                           'snake': to_snakecase,
                           'space': to_spacecase}
 
+    _language_naming_convention = {'.java': 'pascal',
+                                   '.js': 'kebab',
+                                   '.py': 'snake'}
```

Comment:
I would allow it for this merge, but we should definitely introduce a simple system somewhere else mapping file endings to language identifiers, so we have a single interface for that kind of functionality :+1: